components/Common/AvatarGroup/avatar/index.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { userNameToAvatarUrl } from '@/util/github';
+import Avatar from './';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof Avatar>;
+type Meta = MetaObj<typeof Avatar>;
+export const Default: Story = {
```

Comment:
Missing empty line to separate types

Reply:
Fix, out of date because we are using RadixUI.